applications/chain-extension-suite.md

```diff
@@ -0,0 +1,240 @@
+
+
+```rust
+
+/// Sample of the final PSP22 code …. 
+impl PSP22 for AssetContract {
+
+  /// Transfer tokens from the caller to the `to` account.
+  #[ink(message)]
+  fn transfer(
+    &mut self, 
+    to: AccountId, 
+    value: Balance,
+    _data: Vec<u8>
+  ) -> Result<(), PSP22Error> {
+     let result = self
+       .env()
+       .extension()
+       .transfer(self.asset_id, to, value)
+       .map_err(PSP22Error::from);
+    
+     if result.is_ok() {
+       self.env().emit_event(Transfer {
+         from: Some(self.env().caller()),
+         to: Some(to),
+        value,
+       });
+     }
+     result
+  }
+}
+```
+
+### ink! v5 Upgrade
+
+The forthcoming release of ink! v5 is anticipated to bring about enhanced features and optimizations. Being at the forefront of these developments, the Chain Extension Suite is designed to be compatible with ink! v5 right out of the gates.
+
+Additionally, this grant encompasses the upgrade of `pallet_hookpoints` to be compatible with ink! v5. This proactive upgrade signifies a commitment to keeping the toolset current and aligned with the latest advancements in the ink! ecosystem.
+
+## Ecosystem Fit
+
+ink! has been carving out its space within the blockchain community as an adept smart contracting language. Its ease of use, coupled with the familiar Rust domain-specific language (DSL), makes it a favorable choice for developers venturing into the Substrate-based blockchain development.
+
+However, despite its growing popularity, developers often find themselves entangled in the intricacies of low-level communication when integrating ink! into their runtimes. This level of complexity could potentially deter or slow down developers, especially those new to the Substrate ecosystem. The lack of higher-level abstractions and streamlined tooling often necessitates a deeper dive into the underlying communication protocols between the runtime and smart contracts, which could be quite daunting.
+
+Addressing this gap, the Chain Extension Suite, in harmony with the [Hookpoints Suite](https://github.com/deep-ink-ventures/pallet_hookpoints), is envisioned to significantly ease the developer's journey. By abstracting away the low-level communication and boilerplate code, these suites allow developers to focus more on the business logic and less on the underlying plumbing. The auto-generating CLI tool within the Chain Extension Suite is particularly aimed at simplifying the process of exposing runtime methods to smart contracts, thereby accelerating the development process. Together, these suites aim to foster a more developer-friendly ecosystem, enabling developers to kickstart their projects swiftly, and contribute to the flourishing Substrate ecosystem with lesser hurdles.
+
+
+## Team
+
+### Contact
+
+- **Contact Name:** Jakob Drzazga
+- **Contact Email:** 	admin@deep-ink.ventures
```

Comment:
Despite having 3 FTE, there is only 1 contact/team member listed. Could you list the 2 remaining members?